bricks-model/src/main/java/com/github/ingogriebsch/bricks/model/lombok.config

```diff
@@ -0,0 +1,2 @@
+lombok.fieldDefaults.defaultPrivate = true
+lombok.fieldDefaults.defaultFinal = false
```

Comment:
Whole file should be moved to src/main/resources.

Reply:
you think? it is code, rather than a resource, isn't it?

Reply:
IMO not. It's a configuration which helps us to generate some code, okay. But it is itself no code at all, therefore should be not put into the java folder. Additionally I would always assume to search for a configuration file in the resource folder.